examples/h2o/wine-quality.csv

```diff
@@ -0,0 +1 @@
+../resources/wine-quality.csv
```

Comment:
I think it might be better if each example had its own copy of the file so that it's self-contained, instead of putting the file in "resources". Otherwise if you just copy an example folder (a natural way to start your own project), it will suddenly stop working.

Reply:
done